test/parallel/test-http2-respond-with-fd-errors.js

```diff
@@ -47,7 +47,7 @@ const tests = specificTests.concat(genericTests);
 let currentError;
 
 // mock respondFD because we only care about testing error handling
-Http2Stream.prototype.respondFD = () => currentError.ngError;
+Http2Stream.prototype.respond = () => currentError.ngError;
```

Comment:
the comment above would need to be updated as well.

Reply:
@addaleax ^

Reply:
@mcollina Ah, right. Done! :)